lib/src/models/prayer_calculation_settings.dart

```diff
@@ -48,44 +55,56 @@ abstract class PrayerCalculationSettings implements Built<PrayerCalculationSetti
   // ----------------------------- SERIALIZER -----------------------------
 
   /// [Serializer] for this object.
-  static Serializer<PrayerCalculationSettings> get serializer => _$prayerCalculationSettingsSerializer;
+  static Serializer<PrayerCalculationSettings> get serializer =>
+      _$prayerCalculationSettingsSerializer;
 
   // ----------------------------- PROPERTIES -----------------------------
 
   /// Calculation parameter for imsak prayer.
+  @nullable
```

Comment:
Thanks for your contribution @mumer92! These properties should not be marked as nullable IMO as the codes which consume will always assume that these properties are not null. Marking these as nullable may causes instability in library consumer.

Reply:
Hi,

I was testing your library it was not working. 
`$ flutter doctor

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, v0.9.4, on Mac OS X 10.14.1 18B75, locale en-PK)
[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.0)
[✓] Android Studio (version 3.1)
[✓] IntelliJ IDEA Community Edition (version 2017.2.7)
[✓] VS Code (version 1.28.2)
[✓] Connected devices (1 available)

• No issues found!
`

I downloaded project and tried building it from source. But was not able to build and run example. Apparently the original error was `mark field nullable` So I looked up and made this change. Please test and review.